docs/sources/dashboards/share-dashboards-panels/_index.md

```diff
@@ -0,0 +1,372 @@
+1. Do one or both of the following:
+   - Click **Copy link**.
+   - Click **Render image**, which [renders the panel as a PNG image](ref:image-rendering).
+1. Send the copied link or image URL to a Grafana user with authorization to view it.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+#### Query string parameters for server-side rendered images
+
+When you click **Render image** in the panel link settings, Grafana generates a PNG image of the panel with the following default parameters:
+
+| Parameter | Description                                                                                                                                                |
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| width     | Width in pixels. Default is 800.                                                                                                                           |
+| height    | Height in pixels. Default is 400.                                                                                                                          |
+| tz        | Timezone in the format `UTC%2BHH%3AMM` where HH and MM are offset in hours and minutes after UTC.                                                          |
+| timeout   | Number of seconds. The timeout can be increased if the query for the panel needs more than the default 30 seconds.                                         |
+| scale     | Numeric value to configure device scale factor. Default is 1. Use a higher value to produce more detailed images (higher DPI). Supported in Grafana v7.0+. |
+
+You can also update these parameters in the [image rendering configuration](ref:image-rendering-config).
+
+The following example shows a link to a server-side rendered PNG:
+
+```bash
+https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&from=1568719680173&to=1568726880174&panelId=4&fullscreen
+```
+
+<!-- above grafana play link isn't valid but when i try to generate one, the URL is very long -->
+
+### Share an embed
+
+You can share a panel by embedding it on another website using an iframe. Users must be signed into Grafana to view the panel.
+
+{{< admonition type="note" >}}
+As of Grafana 8.0, anonymous access permission is no longer available for Grafana Cloud.
+{{< /admonition >}}
+
+To create a panel that can be embedded, follow these steps:
+
+1. Hover over any part of the panel you want to share to display the actions menu on the top-right corner.
+1. Click the menu and select **Share embed**.
+
+   The **Share embed** drawer opens.
+
+1. (Optional) Toggle the **Lock time range** switch to set whether the panel uses the current relative time range or an absolute time range. This option is enabled by default.
+1. Select the theme for the dashboard. Choose from **Current**, **Dark**, or **Light**.
```

Comment:
Can you make changes to the HTML at this step?

Reply:
Yes, the user can make changes in the editor and then copy the result.